MeshLib/Node.h

```diff
@@ -37,14 +31,6 @@ class Element;
  */
 class Node : public MathLib::Point3dWithID
 {
-	/* friend functions: */
-	friend bool MeshLayerMapper::layerMapping(MeshLib::Mesh &mesh, const GeoLib::Raster &raster, double noDataReplacementValue);
-	friend MeshLib::Mesh* MeshSurfaceExtraction::getMeshSurface(const MeshLib::Mesh &mesh, const MathLib::Vector3 &dir, double angle, bool keep3dMeshIds);
```

Comment:
yes they work because I changed non-public functions to public. let me know if you have any concern about it.